python/cudf/cudf/_lib/parquet.pyx

```diff
@@ -87,3 +95,53 @@ cpdef read_parquet(filepath_or_buffer, columns=None, row_group=None,
         df.index.name = new_index_name
 
     return df
+
+cpdef write_parquet(
+        _Table table,
+        path,
+        compression=None,
+        statistics="ROWGROUP"):
+    """
+    Cython function to call into libcudf API, see `write_parquet`.
+
+    See Also
+    --------
+    cudf.io.parquet.write_parquet
+    """
+
+    # Create the write options
+    cdef string filepath = <string>str(path).encode()
+    cdef sink_info sink = sink_info(filepath)
+    cdef table_metadata *tbl_meta = new table_metadata()
+
+    cdef compression_type comp_type
+    if compression is None:
+        comp_type = compression_type.none
+    elif compression == "snappy":
+        comp_type = compression_type.snappy
+    else:
+        raise ValueError("Unsupported `compression` type")
+
+    cdef statistics_freq stat_freq
+    statistics = statistics.upper()
+    if statistics == "NONE":
+        stat_freq = statistics_freq.STATISTICS_NONE
+    elif statistics == "ROWGROUP":
+        stat_freq = statistics_freq.STATISTICS_ROWGROUP
+    elif statistics == "PAGE":
+        stat_freq = statistics_freq.STATISTICS_PAGE
+    else:
+        raise ValueError("Unsupported `statistics_freq` type")
+
+    cdef write_parquet_args args = write_parquet_args(sink,
+                                                      table.view(),
```

Comment:
Should we get the table view first and then construct this in the `nogil` block?